src/fontra/views/fontinfo/panel-axes.js

```diff
@@ -15,44 +15,45 @@ import { piecewiseLinearMap } from "../core/var-model.js";
 import { IconButton } from "../web-components/icon-button.js"; // for <icon-button>
 import { UIList } from "../web-components/ui-list.js";
 import { BaseInfoPanel } from "./panel-base.js";
+import { translate } from "/core/localization.js";
 import "/web-components/add-remove-buttons.js";
 import { dialogSetup } from "/web-components/modal-dialog.js";
 
 const presetAxes = [
   {
-    label: "Weight",
```

Comment:
I'm not sure what to do instead, but these are not correct: the "Weight" label is not (just) a UI label, but it is a "label" attribute for the axis to be created.

For example, when choosing optical size:

<img width="524" alt="image" src="https://github.com/googlefonts/fontra/assets/4246121/84e290e6-de58-42bf-aa81-2a8a279bfb96">

The last (editable) field should read "Optical Size", not "optical-size".